index.js

```diff
@@ -1,8 +1,15 @@
-function pipeLogsToTerminal() {
+function pipeLogsToTerminal(config) {
   let oldConsoleWarn;
   let oldConsoleError;
   let logs = [];
 
+  Cypress.on('fail', error => {
```

Comment:
I added this because when the failed command is in pending state. So log still labeled it as passed (green). This is to fix this problem.